packages/shared/lib/core/app/interfaces/evm-address.interface.ts

```diff
@@ -1,4 +1,4 @@
-export interface IEvmAddressEvent {
+export interface IEvmAddress {
```

Comment:
IMO the `event` suffix makes it more clear that this is only the returned event when we receive it